admin/sql/updates/2019-01-04-add-user-login-id.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+-- Add column for alternative user id for login purpose to user table
+ALTER TABLE "user" ADD COLUMN user_login_id UUID NOT NULL DEFAULT uuid_generate_v4();
```

Comment:
I just noticed that there's no index on this field, there should be if we use it for lookups